Source/ManagedNativeWifi/ManagedNativeWifi.csproj

```diff
@@ -1,17 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>

```

Comment:
Again, as for .NET Standard, do you have any particular reason to specify 2.1?
It is because if it is 2.0, it can cover .NET Framework 4.6.2 to 4.8 so that we can decrease the number of targets when we stop supporting .NET 4.5.2 to 4.6 in future.